### PR TITLE
[atoms] use css locators in dom.js

### PR DESCRIPTION
**Pull request by joerg1985**
### Description
Using xpaths will need to transfer the complete `wicked-good-xpath`, this can easily be replaced by a css locator.

### Motivation and Context
This will reduce the size of the `isDisplayed.js` from ~43kb to ~17kb.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by titusfortner** [comment]
@twalpole I know you did some work on a separate isDisplayed atom for Capybara. Did you do something similar here?

**Comment by twalpole** [comment]
@titusfortner I just went through all the code and removed all the stuff that was supporting legacy browsers - which took it down to 4.7k - [lib/capybara/selenium/atoms/isDisplayed.min.js](https://github.com/teamcapybara/capybara/blob/master/lib/capybara/selenium/atoms/isDisplayed.min.js) - unminified 16.1k https://github.com/teamcapybara/capybara/blob/master/lib/capybara/selenium/atoms/src/isDisplayed.js.  I did not specifically swap from xpath to css because of the added scoping functionality xpath provides, but it likely did involve getting rid of wicked-good-xpath since every modern browser supports it natively without needing a JS polyfill.

File: javascript/atoms/BUILD.bazel [hunk 1]
```diff
@@ -250,11 +250,12 @@ closure_js_library(
     ],
     deps = [
         ":bot",
+        ":css",
         ":dom",
         ":errors",
         ":json",
         ":useragent",
-        ":xpath",
+        "//third_party/js/wgxpath",
```
Comment (titusfortner) [inline javascript/atoms/BUILD.bazel:258]: So, do we need to depend on wicked good xpath here?

**Comment by titusfortner** [comment]
We've also discussed [moving from closure to Typescript](https://github.com/SeleniumHQ/selenium/issues/9668#issuecomment-1072652972) for these.

File: javascript/atoms/BUILD.bazel (hunk 1 above)
Comment (joerg1985) [reply javascript/atoms/BUILD.bazel:258]: Yes, this is still needed here for legacy browsers to have XPath locator support.

This PR does only remove the need for wicked good xpath for is-displayed atom.

**Comment by joerg1985** [comment]
> We've also discussed [moving from closure to Typescript](https://github.com/SeleniumHQ/selenium/issues/9668#issuecomment-1072652972) for these.

Is it possible to run TypeScript in IE 11 and old Android versions?

File: javascript/atoms/BUILD.bazel (hunk 1 above)
Comment (titusfortner) [reply javascript/atoms/BUILD.bazel:258]: Which legacy browsers require it? IE 11?

**Comment by titusfortner** [comment]
@joerg1985

> Is it possible to run TypeScript in IE 11 and old Android versions

Wouldn't we transpile and minify it the same (ish) way closure does?

**Review by diemol — APPROVED** [review]
Thank you, @joerg1985!

**Pull request merged**